### PR TITLE
fix: emui build

### PR DESCRIPTION
**Pull request by Dartoxian**
## Description:
This PR brings the emui into the ci build/release process. It is a reversion of last nights https://github.com/kurtosis-tech/kurtosis/pull/1813.

Yesterday the build/release change for emui was abandoned because after several attempts the publish step was not working. In https://github.com/kurtosis-tech/kurtosis/commit/4ed9328ab2743c045a7dfa71c8c4e7d4122aae35 I have demonstrated that this pr works by:
* Removing the body of the publish jobs and replacing them with echo statements
* Triggering publish jobs on `bgazzard-*` tags
* Pushing a tag `bgazzard-0.0.1` (which I have now deleted) to trigger the `publish` workflow
* The publish workflow can be seen here https://app.circleci.com/pipelines/github/kurtosis-tech/kurtosis/9407/workflows/ca6cef3b-4f71-481a-ab08-8787002ed9d0 and demonstrates that the `build_enclave_manager_webapp` job is now successfully run as a dependency of `publish_engine_server_image`.

The existing build/test path which was approved previously is unchanged by this pr and can be seen working in this workflow https://app.circleci.com/pipelines/github/kurtosis-tech/kurtosis/9408/workflows/bb48e469-2ff7-433e-8da9-ad32620b0bbc.

## Is this change user facing?
NO

### CODE REVIEW

**Comment by Dartoxian** [comment]
Marked as do not merge as I expect we want to hold off on this until Tuesday.

File: enclave-manager/web/package.json [hunk 1]
```diff
@@ -31,8 +31,6 @@
     "analyze": "lerna run analyze",
     "cleanInstall": "rm -rf node_modules; rm -rf packages/*/node_modules; yarn install; lerna init",
     "clean": "lerna run clean",
-    "prebuild": "rm -rf ../../engine/server/webapp",
-    "postbuild": "cp -r packages/app/build/ ../../engine/server/webapp",
```
Comment (adschwartz) [inline enclave-manager/web/package.json]: @Dartoxian won't you still need this if you want to build a development engine and push it to a cloud instance (like from your laptop)?

Comment (adschwartz) [reply enclave-manager/web/package.json]: Also another thing to think about, if you run the main (root) build script will that include the EM UI? It needs to, especially if you want to test locally before pushing and eventually making a PR.

Comment (Dartoxian) [reply enclave-manager/web/package.json]: yep thanks for catching this @adschwartz - I've updated the pr and tested `scripts/build.sh` is producing a working `kurtosis web` locally (and validated that it wasn't prior to this update).

**Review by adschwartz — APPROVED** [review]

**Pull request merged**